t3f/ops.py

```diff
@@ -839,8 +849,10 @@ def frobenius_norm(tt, epsilon=1e-5, differentiable=False):
       a fast and stable implementation based on QR decomposition.
 
   Returns
-    a number
-    sqrt of the sum of squares of all elements in `tt`
+    a number which is the Frobenius norm of `tt`, if it is `TensorTrain`
```

Comment:
Please, add tensortrainbatch version into args description and the first line of the docsrting